modes/parkour/interface.lua

```diff
@@ -346,7 +346,7 @@ onEvent("GameDataLoaded", function(data)
 			end
 
 			webhooks._count = webhooks._count + 1
-			webhooks[webhooks._count] = "**[BANS]** __" .. action[3] .. "__ has " .. action[1] .. "ned the player with ID __" .. action[2] .. "__."
+			webhooks[webhooks._count] = "**`[BANS]:`** **" .. action[3] .. "** has " .. action[1] .. "ned a player. (PID: **" .. action[2] .. "**)"
```

Comment:
PID is not ID, there's a big difference in transformice (mostly when working with bots)

Reply:
Just to tell you, PID is more like a temporal ID than a static one. It sets you a new one everytime you connect to the game. We're using the static ID here.